app/views/dashboard/sub_service_requests/index.json.jbuilder

```diff
@@ -0,0 +1,7 @@
+json.(@sub_service_requests) do |ssr|
```

Comment:
The file itself isn't doing this. For some reason my github is setting tabs to 8 spaces